tasks/pipeline.py

```diff
@@ -1029,11 +1029,16 @@ def compare_to_itself(ctx):
     # Push an empty commit to prevent linking this pipeline to the actual PR
     ctx.run("git commit -m 'Compare to itself' --allow-empty", hide=True)
     ctx.run(f"git push origin {new_branch}")
+
+    from tasks.libs.releasing.json import load_release_json
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Could you import this at the top of the function?

Reply:
From the description:

> Additional Notes
I had to lazily load the function to be able to mock it properly. Not great but I don't think it would be wise to modify the tests too much in this PR